src/renderer/selfkey-id/main/components/selfkey-id-applications.jsx

```diff
@@ -261,9 +271,254 @@ const getRpInfo = (rpName, field) => {
 const MARKETPLACE_ROOT_PATH = '/main/marketplace-categories';
 
 class SelfkeyIdApplicationsComponent extends Component {
```

Comment:
@rodrigopavezi are you sure this is right? I wouldnt expect ui to be only on release branch maybe it is overriding some changes we did in dev branch?

Reply:
@sk91 that is correct. It was only changed on the release branch and there was no merging issue when I merge dev into this branch.